packages/flame/test/components/isometric_tile_map_component_test.dart

```diff
@@ -24,5 +27,15 @@ void main() {
         expect(c.getBlock(c.getBlockCenterPosition(block)), block);
       }
     });
+    test('center position', () {
+      final c = IsometricTileMapComponent(
+        _MockSpriteSheet(),
+        map,
+        destTileSize: Vector2.all(32.0),
+        tileHeight: 8.0,
+      );
+
+      expectVector2(c.getBlockCenterPosition(const Block(0, 0)), Vector2(0, 0));
```

Comment:
I could check other values but their are ugly and not intuitive, so I think a test won't help at all.